README.md

```diff
@@ -2,4 +2,4 @@
 
 This repository contains the latest changes to the BitGo API reference published on the [Developer Portal](https://developers.bitgo.com/api/overview).
 
-You can view the entire OpenAPI 3.0 specification in [api.yaml](https://github.com/BitGo/api-changelog/blob/master/api.yaml). To view individual updates, see the [commits](https://github.com/BitGo/api-changelog/commits/master/api.yaml) history.
+You can view the entire OpenAPI 3.0 specification in [api.yaml](https://github.com/BitGo/api-changelog/blob/master/api.yaml). To view individual updates, see [Releases](https://github.com/BitGo/api-changelog/releases) or the [commits](https://github.com/BitGo/api-changelog/commits/master/api.yaml) history.
```

Comment:
```suggestion
You can view the entire OpenAPI 3.0 specification in [api.yaml](https://github.com/BitGo/api-changelog/blob/master/api.yaml). To view individual updates, see [Releases](https://github.com/BitGo/api-changelog/releases) or the [Commits](https://github.com/BitGo/api-changelog/commits/master/api.yaml) history.
```

Let's either capitalize both or lower case both. I think the capitalization works